client/landing/stepper/declarative-flow/domain-transfer.ts

```diff
@@ -56,14 +56,11 @@ const domainTransfer: Flow = {
 
 	useStepNavigation( _currentStepSlug, navigate ) {
 		const flowName = this.name;
-		const { setStepProgress } = useDispatch( ONBOARD_STORE );
-		const flowProgress = useFlowProgress( { stepName: _currentStepSlug, flowName } );
 		const userIsLoggedIn = useSelect(
 			( select ) => ( select( USER_STORE ) as UserSelect ).isCurrentUserLoggedIn(),
 			[]
 		);
 		const locale = useLocale();
-		setStepProgress( flowProgress );
```

Comment:
Is this still necessary maybe? It looks like removing it potentially [breaks a tracks event](https://github.com/Automattic/wp-calypso/blob/eab10b490153bc5be1b2fd750c55d7bac5ff606e/client/landing/stepper/declarative-flow/internals/index.tsx#L131C2-L131C2) but other than that I can't figure out why we need it (Aside from knowing to show the progress bar that is)

Reply:
Thanks for sharing your investigation thoughts.
Comparing to production, we're firing the same events, so we should be fine here. I also don't see any application for `setStepProgress` other than progress bar.